axelor-sale/src/main/resources/views/Partner.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+			  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+			  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.2.xsd">
+
+	<form name="partner-customer-form" title="Customer" model="com.axelor.apps.base.db.Partner"
```

Comment:
partner-customer-form already exists in base module. It should work if you just delete the view in base module.

Reply:
but we got an id here
id="partner-customer-sales-form"